src/test/java/marquez/api/DatasourceIntegrationTest.java

```diff
@@ -0,0 +1,31 @@
+package marquez.api;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.junit.Assert.assertEquals;
+
+import java.net.URI;
+import javax.ws.rs.core.MediaType;
+import javax.ws.rs.core.Response;
+import marquez.api.models.DatasourcesResponse;
+import marquez.db.fixtures.AppWithPostgresRule;
+import org.junit.ClassRule;
+import org.junit.Test;
+
+public class DatasourceIntegrationTest {
```

Comment:
Can you provide a unit test for `DatasourceResource` first?

Reply:
Please see PRs #394, #387, and #384, where this is fixed.